### PR TITLE
feat(core): add web only filter, backdrop, mix-blend-mode

### PR DESCRIPTION
**Pull request by ehxxn**
add the following props support ( web only )
`filter`
`backdropFilter`
`mixMlendMode`

### CODE REVIEW

File: packages/web/src/interfaces/RNExclusiveTypes.ts [hunk 1]
```diff
@@ -19,12 +24,15 @@ export interface RNExtraProps {
...
+export interface RNViewProps
+  extends GestureResponderHandlers,
+    RNExtraProps,
+    WebOnlyProps {
   rel?: any
   download?: any
 }
 
-export interface RNTextProps extends RNExtraProps {
+export interface RNTextProps extends RNExtraProps, WebOnlyProps {
```
Comment (natew) [inline packages/web/src/interfaces/RNExclusiveTypes.ts:35]: i think this is wrong, should only be part of style props

File: packages/web/src/interfaces/RNExclusiveTypes.ts [hunk 2]
```diff
@@ -2,6 +2,11 @@ import type { GestureResponderHandlers, LayoutChangeEvent } from 'react-native'
 
 type OnLayout = ((event: LayoutChangeEvent) => void) | undefined
 
+export interface WebOnlyProps {
+  filter?: React.CSSProperties['filter']
+  backdropFilter?: React.CSSProperties['backdropFilter']
+  mixBlendMode?: React.CSSProperties['mixBlendMode']
+}
```
Comment (natew) [inline packages/web/src/interfaces/RNExclusiveTypes.ts:9]: i think we have a web only style props interface somewhere in web/types

File: packages/helpers/src/validStyleProps.ts [hunk 3]
```diff
@@ -149,6 +149,11 @@ export const stylePropsView = {
   justifyContent: true,
   left: true,
   margin: true,
+  ...(isWeb && {
```
Comment (natew) [inline packages/helpers/src/validStyleProps.ts:152]: there's a place for this already just below

**Comment by natew** [comment]
we need to update the docs for these too in styles.mdx. if we're updating these, we should be more thorough and catalog which are important and at least plausibly likely to be implemented by react native add a few more.

**Comment by TheEhsanSarshar** [comment]
> we need to update the docs for these too in styles.mdx. if we're updating these, we should be more thorough and catalog which are important and at least plausibly likely to be implemented by react native add a few more.

we can also have these in react-native, there is a [library](https://github.com/iyegoroff/react-native-color-matrix-image-filters#supported-filters) for that. but for now we can skip it

**Comment by TheEhsanSarshar** [comment]
@natew Added 30 more style props
I think it's better to have a separate PR to make current Apis compatible with web and deprecate old ones.
and keep this PR just adding new web styles

File: packages/web/src/types.tsx [hunk 4]
```diff
@@ -1320,6 +1316,204 @@ interface ExtraStyleProps {
     | 'bottom'
     | TwoValueTransformOrigin
     | `${TwoValueTransformOrigin} ${Px}`
+  /**
+   * @platform: web
```
Comment (natew) [inline packages/web/src/types.tsx:1320]: Can we make these say like the others ` Web-only style property. Will be omitted on native.`

File: packages/web/src/constants/webToNativeProps.native.tsx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+      contain: 'contain',
+      cover: 'cover',
+    }
+
+    let resizeMode = resizeModeMap[val] || 'cover'
+
+    return [['resizeMode', resizeMode]]
+  },
+  verticalAlign: (val: any) => {
+    const data = {
```
Comment (natew) [inline packages/web/src/constants/webToNativeProps.native.tsx:16]: hoist this out of the function to be faster

File: packages/web/src/constants/webToNativeProps.native.tsx [hunk 6]
```diff
@@ -0,0 +1,88 @@
+export const webToNativeDynamicExpansion = {
+  objectFit: (val: any) => {
+    const resizeModeMap = {
```
Comment (natew) [inline packages/web/src/constants/webToNativeProps.native.tsx:3]: hoist this out of the function to be faster

File: apps/site/components/HeaderMenu.tsx [hunk 7]
```diff
@@ -1,7 +1,8 @@
 import { ThemeTintAlt, useTint } from '@tamagui/logo'
 import { Menu } from '@tamagui/lucide-icons'
 import * as React from 'react'
-import type { ThemeName } from 'tamagui'
+import type {
```
Comment (natew) [inline apps/site/components/HeaderMenu.tsx:4]: formatting

File: packages/web/types/types.d.ts [hunk 8]
```diff
@@ -738,6 +734,204 @@ interface ExtraStyleProps {
...
+    paddingBlockEnd?: SpaceTokens | number;
+    paddingInline?: SpaceTokens | number;
+    paddingInlineStart?: SpaceTokens | number;
+    paddingInlineEnd?: SpaceTokens | number;
+    objectFit?: Properties['objectFit'];
+    verticalAlign?: Properties['verticalAlign'];
+    inset?: SpaceTokens | number;
+    insetBlock?: SpaceTokens | number;
+    insetBlockStart?: SpaceTokens | number;
+    insetBlockEnd?: SpaceTokens | number;
```
Comment (natew) [inline packages/web/types/types.d.ts:925]: are these supported on native?

Comment (TheEhsanSarshar) [reply packages/web/types/types.d.ts:925]: yeah, they are mapped to native alternatives.
like `paddingInline` to `paddingHorizontal`

**Pull request merged**